tests/SwaggerProvider.ProviderTests/v3/Swagger.I0219.Tests.fs

```diff
@@ -8,3 +8,9 @@ let Schema = __SOURCE_DIRECTORY__ + "/../Schemas/v3/issue219.yaml"
 type AcmeApi = OpenApiClientProvider<Schema>
 
 let inst = AcmeApi.Client()
+
+let askDetailRequest = new AcmeApi.AskDetailRequest(11)
+
+let typedTask = inst.AskDetail(askDetailRequest: AcmeApi.AskDetailRequest)
+
+typedTask.GetType().FullName.Contains("AskDetailResponse")
```

Comment:
It should be proper xUnit test - https://github.com/fsprojects/SwaggerProvider/blob/188bdfb22db9e055526559796f0cba7751ad7928/tests/SwaggerProvider.ProviderTests/v3/Swagger.PetStore.Tests.fs#L23

I assume it fails now, right?